src/patterns/error-status-pages/index.njk

```diff
@@ -164,27 +164,36 @@ The page should have:
     patternlibExample({ "path": "patterns/error-status-pages/examples/503-error/index.njk" })
 }}
 
-## Timeout
+## Session timed out error
 
-Use a Timeout page when the user has been timed out of the service due to inactivity.  Explain why the timeout has occured and reassure the user that their progress has been saved. Provide a link to allow them to sign in again.
+Use a timeout error if you need to store the user's information in a browser session with an expiry time to protect their information. When the session expires, use the timeout error message.
 
-The page should have:
+Timeout could be needed because a user has been inactive, or because they have followed a link to the middle of a process where the required session data is missing.
```

Comment:
```suggestion
This error could be needed because a user has been inactive, or because they have followed a link to the middle of a process where the required session data is missing.
```